packages/remix-dev/config.ts

```diff
@@ -371,9 +370,6 @@ export async function readConfig(
     case "cloudflare-pages":
       serverBuildPath = "functions/[[path]].js";
       break;
-    case "netlify":
-      serverBuildPath = ".netlify/functions-internal/server.js";
-      break;
```

Comment:
Removing this would be a breaking change for existing projects I think, so I think we should keep it (for now)

```suggestion
    case "netlify":
      serverBuildPath = ".netlify/functions-internal/server.js";
      break;
```

Reply:
Yeah, I'm aware. Just sorting out some stuff in regards to this.

Reply:
Done